client/admin/components/Program/Enrollments/EnrollmentDialog.vue

```diff
@@ -59,27 +61,28 @@ export default {
     isLoading: false
   }),
   methods: {
-    enroll() {
-      enrollmentApi.create(pick(this, ['learnerId', 'programId'])).then(() => {
-        this.close();
-        this.$emit('enrolled');
-      });
+    async enroll() {
+      const params = pick(this, ['learnerId', 'programId']);
+      await enrollmentApi.create(params);
+      this.close();
+      this.$emit('enrolled');
     },
     close() {
       this.visible = false;
       this.learnerId = null;
     },
+    processLearners({ items: learners }) {
```

Comment:
Similar as prev, if you are setting something on the context prefix with set or similar. Not sure it makes sense to extract it when method is rather small and not reused (same for prev)